IGC/Compiler/Optimizer/OpenCLPasses/AlignmentAnalysis/AlignmentAnalysis.cpp

```diff
@@ -165,7 +165,11 @@ unsigned int AlignmentAnalysis::visitSelectInst(SelectInst& I)
 
 unsigned int AlignmentAnalysis::visitPHINode(PHINode& I)
 {
+#if LLVM_VERSION_MAJOR >= 14
+    uint64_t newAlign = Value::MaximumAlignment;
+#else
     unsigned int newAlign = Value::MaximumAlignment;
+#endif
```

Comment:
You can use `auto` here, however it's good to make a quick code check and make sure there are no obvious problems. Somethimes `auto` can behave unexpectedly.